src/Umbraco.Web.Common/Extensions/UmbracoCoreServiceCollectionExtensions.cs

```diff
@@ -24,14 +25,65 @@
 using Umbraco.Core.Logging;
 using Umbraco.Core.Logging.Serilog;
 using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.SqlSyntax;
 using Umbraco.Core.Runtime;
 using Umbraco.Web.Common.AspNetCore;
 using Umbraco.Web.Common.Runtime.Profiler;
 
-namespace Umbraco.Web.Common.Extensions
+namespace Umbraco.Extensions
 {
     public static class UmbracoCoreServiceCollectionExtensions
     {
+        public static IServiceCollection AddUmbracoSqlCeSupport(this IServiceCollection services)
+        {
+            try
+            {
+                var binFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+                if (binFolder != null)
+                {
+                    var dllPath = Path.Combine(binFolder, "Umbraco.Persistance.SqlCe.dll");
+                    Assembly.LoadFrom(dllPath);
+
+
+                    var sqlCeSyntaxProviderType = Type.GetType("Umbraco.Persistance.SqlCe.SqlCeSyntaxProvider, Umbraco.Persistance.SqlCe");
```

Comment:
We should also use Assembly.Load("...").GetType("...") if this ever returns null which is possible.

Reply:
In this case Assembly.Load from returns the assembly that was loaded into the appdomain so to be safe its changed to use `assembly.GetType`. I've updated.